modules/azure/function_app_windows/main.tf

```
@@ -32,6 +32,7 @@ resource "azurerm_windows_function_app" "function_app" {
     always_on              = var.always_on
     vnet_route_all_enabled = var.route_all_outbound_traffic
     use_32_bit_worker      = var.use_32_bit_worker
+    app_scale_limit        = var.app_scale_limit == null ? 0 : var.app_scale_limit
```

This is just another default value but now more strange....
The only change is that the default has changed from `1` to `0`.

What does the code do if `app_scale_limit` is set to `null` here?

If the value when run that is `0` then please change to
```suggestion
    app_scale_limit        = var.app_scale_limit
```
and the remove the `nullable = true` line from the variable

---

its updated without nullable plus default value set to 0 as null is not possible and resource does not accept it but fails. But 0 make the necessary trick I believe